docs/test.html

```diff
@@ -56,16 +57,14 @@ <h1>Tests</h1>
     publicSuffixList.parse(text, punycode.toASCII);
     stdout('JS');
     checkAll();
-    publicSuffixList.enableWASM().then(status => {
+    /*publicSuffixList.enableWASM().then(status => {
```

Comment:
I'm commenting this out for now. I still don't think it should be so difficult for calling code to enable Wasm.